### ISSUE TITLE
Pugna interrupting channeling items when blinking further than 900 units away.

### ISSUE THREAD

**Issue by EndRay** [ValveSoftware/Dota2-Gameplay#18626]
### Ability name

Oblivion Savant

### Description

Way to replicate:
Create pugna, get veil of discord, blink and tp scroll.
Ctrl+right click on veil to show radius 900.
Tp somewhere.
When blinking inside radius of 900 tp isn't interrupted as intended. When blinking outside - interrupted.
(works the same way when channeling meteor hammer or life drain, even if after cast enemy is still in the radius of life drain)

### Example Match ID (and possibly Timestamp)

N/A

**Comment by EndRay** [comment]
Yes, as I wrote, seems like blink doesn't care about actual cast range of the spell, but checks for how far pugna blinked from his own place, and if he blinked further than 900 units it interrupts even when character stays in the cast range of the channeling skill/item.

**Comment by dampf1** [comment]
It is all about the penalty range for overblinking. If you blink further than the penalty range then the channel will always be broken. The penalty range depends on your added castrange and arcane blink also has an increased range. For the basic blink dagger it starts at 960 range. Does it make sense that it is this way? No.
